src/h5web/visualizations/line/hooks.ts

```diff
@@ -6,23 +6,14 @@ import { findDomain, getAxisScale } from '../shared/utils';
 import { AxisSystemContext } from '../shared/AxisSystemProvider';
 import { useVisProps } from '../../dataset-visualizer/VisProvider';
 
-export function useData(): number[] {
-  const { values, slicingIndices } = useVisProps();
-
-  if (slicingIndices) {
-    return (values as number[][])[slicingIndices[0]];
-  }
-
-  return values as number[];
-}
-
 export function useDataDomain(): Domain | undefined {
-  const data = useData();
-  return useMemo(() => findDomain(data), [data]);
+  const { values } = useVisProps();
+  return useMemo(() => findDomain(values.data as number[]), [values.data]);
 }
 
 export function useDataPoints(): Vector3[] | undefined {
-  const data = useData();
+  const { values } = useVisProps();
+  const data = values.data as number[];
```

Comment:
I have to dig into the typing of `values.data` to see if the casting here can be removed/replaced